stock_picking_show_move_lots/i18n/es.po

```diff
@@ -0,0 +1,21 @@
+# Translation of Odoo Server.
```

Comment:
Catalan translation file missing ("Lot/Número de sèrie")

Reply:
They are from aragón :(((((

Reply:
Ignore it then

Reply:
Too late, but dont make me remove it pleass